CMakeLists.txt

```diff
@@ -173,7 +173,7 @@ if (USE_CRYPTO)
 endif ()
 
 if(KVZ_BUILD_SHARED_LIBS)
-  list( APPEND CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib" "./" "../lib" )  
+  list( APPEND CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}" "./" "../lib" )  
```

Comment:
Are there any cases where `GNUInstallDirs` is broken?

See https://gitlab.kitware.com/cmake/cmake/-/issues/18640 which suggests `$LIB_SUFFIX` is not the way to go...

Reply:
@kmilos you're completely right and I don't know how I could forget about it.